optuna/trial/_trial.py

```diff
@@ -48,12 +44,7 @@ class Trial(BaseTrial):
 
     """
 
-    def __init__(
-        self,
-        study,  # type: Study
-        trial_id,  # type: int
-    ):
-        # type: (...) -> None
+    def __init__(self, study: "optuna.study.Study", trial_id: int,) -> None:
```

Comment:
This doesn't matter but for the consistency...
```suggestion
    def __init__(self, study: "optuna.study.Study", trial_id: int) -> None:
```